File: Python/1672. Richest Customer Wealth.py
class Solution(object):
    def maximumWealth(self, accounts):
        """
        :type accounts: List[List[int]]
        :rtype: int
        """
        MaxWealth = 0
        Wealths= [0] * len(accounts)
        for i in range(len(accounts)):
            for j in range(len(accounts[i])):
                Wealths[i] = Wealths[i] + accounts[i][j]
        
        for i in range(len(Wealths)):
            MaxWealth = max(MaxWealth, Wealths[i])

        return MaxWealth

File: C++/1672. Richest Customer Wealth.cpp
#include <iostream>
#include <vector>

using namespace std;

class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {

        vector<int> wealths(accounts.size(), 0);
        int maxWealth = 0;

        for(int i = 0; i < accounts.size(); i++){
            for(int j=0; j < accounts[i].size(); j++){
                wealths.at(i) = accounts[i][j] + wealths[i];
            }
        }
        for(int i = 0; i < wealths.size(); i++){
            maxWealth = max(maxWealth, wealths[i]);
        }

        return maxWealth;

    }
};

